### DISCUSSION TITLE
Run "gatsby develop" with a specific file of environment variables

### DISCUSSION THREAD

**Discussion by robertocerdasilva** [gatsbyjs/gatsby#33097] [Help]
Hi, I have a question.

I have 2 files with environment variables:
```
.env.development
.env.staging
```

When I execute the command `gatsby develop` by default it takes the value of the environment variables that are in the file ".env.development". That's fine...

The question is: Is it possible through the `gatsby develop` command to consider the value of the environment variables that exist in ".env.staging" and not in ".env.development"?

I need to test in development and currently I have to manually modify the environment variables in ".env.development".

Thanks!!! :D

**Comment by LekoArts** [comment]
Hi!

You could use an environment variable to define this.

So when you run:

```shell
STAGING=true gatsby develop
```

Then `process.env.STAGING` is defined.

In your `gatsby-config.js`:

```js
if (process.env.STAGING) {
  require("dotenv").config({
    path: `.env.staging`,
  })
} else {
  require("dotenv").config({
    path: `.env.${process.env.NODE_ENV}`,
  })
}
```
